app.js

```diff
@@ -31,11 +34,14 @@ mongoose.connect(config.DB_CONNECTION_STRING, {}, (err) => {
 app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
+app.use(validator());
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
+app.use(session());
```

Comment:
Are validator and session things that work hand in hand? If so may want to put validator and session together.